board/pluto/S45msd

```diff
@@ -8,7 +8,7 @@ file=/sys/kernel/config/usb_gadget/composite_gadget/functions/mass_storage.0/lun
 img=/opt/vfat.img
 
 patch_html_page() {
-	LINUX=`uname -a | tr / -`
+	LINUX=`name -a | tr / - | tr '\n' ';' ; echo -n $(nproc) "core(s)"`
```

Comment:
a typo has been introduced ; `uname -> name`

Reply:
Sorry about that - fixed now.